fuzzers/064-gtp-channel-conf/top.py

```diff
@@ -0,0 +1,128 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2017-2020  The Project X-Ray Authors.
+#
+# Use of this source code is governed by a ISC-style
+# license that can be found in the LICENSE file or at
+# https://opensource.org/licenses/ISC
+#
+# SPDX-License-Identifier: ISC
+
+import json
+import os
+import random
+from collections import namedtuple
+
+random.seed(int(os.getenv("SEED"), 16))
+from prjxray import util
+from prjxray import verilog
+from prjxray.db import Database
+
+INT = "INT"
+BIN = "BIN"
+BOOL = "BOOL"
+STR = "STR"
+
+
+def gen_sites():
+    db = Database(util.get_db_root(), util.get_part())
+    grid = db.grid()
+    for tile_name in sorted(grid.tiles()):
+        loc = grid.loc_of_tilename(tile_name)
+        gridinfo = grid.gridinfo_at_loc(loc)
+
+        if gridinfo.tile_type not in [
+                "GTP_CHANNEL_0",
+                "GTP_CHANNEL_0_MID_LEFT",
+        ]:
```

Comment:
I think here we should have all the possible GTP_CHANNEL types and only one is selected. If we allow all the GTP_CHANNEL tiles to be used, we might reduce the number of specimens, as more data is generated in one specimen.

Reply:
I added more types, so in the result we have multiple `GTPE2_CHANNEL` instantiations for different LOCs in a single `top.v` . I reduced number of specimens to 10.